src/main/scala/cromwell/engine/db/package.scala

```diff
@@ -0,0 +1,12 @@
+package cromwell.engine
+
+import cromwell.engine.store.ExecutionStore
+import ExecutionStore.ExecutionStatus.ExecutionStatus
+
+package object db {
+  // CallStatus doesn't exist as an enum already, or reuse ExecutionStatus?
+  type CallStatus = ExecutionStatus
```

Comment:
As per conversation today it sounds like this is superfluous. Let's just use `ExecutionStatus` throughout unless there's a good reason not to.